### PR TITLE
Feature/6.0/wkhtmltopdf

### PR DESCRIPTION
**Pull request by gvollbach**
https://docu.ilias.de/goto_docu_wiki_wpage_5730_1357.html

### CODE REVIEW

File: Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfRenderer.php [hunk 1]
```diff
@@ -0,0 +1,210 @@
+<?php
+
+require_once './Services/PDFGeneration/classes/class.ilPDFGenerationConstants.php';
```
Comment (mjansenDatabay) [inline Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfRenderer.php]: Do we really need the `require_once` directives?

File: Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfConfig.php [hunk 2]
```diff
@@ -0,0 +1,1223 @@
+<?php
+require_once './Services/PDFGeneration/classes/class.ilPDFGenerationConstants.php';
```
Comment (mjansenDatabay) [inline Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfConfig.php]: Do we really need the `require_once` directives?

File: Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfRenderer.php (hunk 1 above)
Comment (gvollbach) [reply Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfRenderer.php]: No, they are still from the plugin, they will be removed after the final cleanup.

File: Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfConfig.php (hunk 2 above)
Comment (gvollbach) [reply Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfConfig.php]: No, they are still from the plugin, they will be removed after the final cleanup.

**Pull request merged**